resources/views/layouts/navbarMobile.blade.php

```diff
@@ -1,6 +1,6 @@
 <div class="flex-grow-1">
   <div id="navbarMobile">
-  <nav class="navbar navbar-light bg-primary d-print-none">
+    <nav class="navbar navbar-light bg-primary d-print-none">
```

Comment:
please check the tag `<nav` is not aligned

Reply:
already checked, it is aligned